frontend/src/components/common/AdminUserTable.tsx

```diff
@@ -19,6 +19,18 @@ interface AdminUserTableProps {
 const AdminUserTable = ({
   adminUsers,
 }: AdminUserTableProps): React.ReactElement => {
+  const [users, setUsers] = useState(adminUsers);
+  // const [order, setOrder] = useState("Ascending");
+  // const sorting = (property: string)=>{
+  //   if (order == "Ascending") {
+  //     const sorted = [...users].sort((a, b)=>
+  //     a[property].toLowerCase() > b[property].toLowerCase() ? 1 : -1
```

Comment:
Over here, I try to parse through the userTypes with their field as a key (firstName, email), but it says a type of string cannot be used to parse. If anyone has any suggestions I'd appreciate it